package.json

```diff
@@ -52,6 +52,7 @@
   "dependencies": {
     "@chainlink/contracts": "^0.0.5",
     "@openzeppelin/contracts": "2.5.1",
+    "@openzeppelin/contracts-ethereum-package": "^2",
```

Comment:
should we use a strict version number?

Reply:
Yea sure I'll update. I believe **3** is already out.